src/main/java/com/easypost/model/Rate.java

```diff
@@ -1,191 +1,407 @@
+     *
+     * @param estDeliveryDays Estimated delivery days for this Rate.
+     */
+    public void setEstDeliveryDays(final Number estDeliveryDays) {
+        this.estDeliveryDays = estDeliveryDays;
+    }
+
+    /**
+     * Get the ID of the shipment of this Rate.
+     *
+     * @return ID of the shipment of this Rate.
+     */
+    public String getShipmentId() {
+        return shipmentId;
+    }
+
+    /**
+     * Set the ID of the shipment of this Rate.
+     *
+     * @param shipmentId ID of the shipment of this Rate.
+     */
+    public void setShipmentId(final String shipmentId) {
+        this.shipmentId = shipmentId;
+    }
+
+    /**
+     * Get the ID of the carrier account of this Rate.
+     *
+     * @return ID of the carrier account of this Rate.
+     */
+    public String getCarrierAccountId() {
+        return carrierAccountId;
+    }
+
+    /**
+     * Set the ID of the carrier account of this Rate.
+     *
+     * @param carrierAccountId ID of the carrier account of this Rate.
+     */
+    public void setCarrierAccountId(final String carrierAccountId) {
+        this.carrierAccountId = carrierAccountId;
+    }
+
+    /**
+     * Get the TimeInTransit of this Rate.
+     *
+     * @return TimeInTransit object.
+     */
+    public TimeInTransit getTimeInTransit() {
         return timeInTransit;
     }
-  public void setTimeInTransit(TimeInTransit timeInTransit) {
+
+    /**
+     * Set the TimeInTransit of this Rate.
+     *
+     * @param timeInTransit TimeInTransit object.
+     */
+    public void setTimeInTransit(final TimeInTransit timeInTransit) {
+        this.timeInTransit = timeInTransit;
+    }
+
+    /**
+     * Create a new Rate object.
+     *
+     * @param id                     ID of this Rate.
+     * @param carrier                The carrier of this Rate.
+     * @param service                The service of this Rate.
+     * @param rate                   The rate of this Rate.
+     * @param currency               The currency of this Rate.
+     * @param listRate               The list rate of this Rate.
+     * @param listCurrency           The list currency of this Rate.
+     * @param retailRate             The retail rate of this Rate.
+     * @param retailCurrency         The retail currency of this Rate.
+     * @param deliveryDays           The delivery days of this Rate.
+     * @param deliveryDate           The delivery date of this Rate.
+     * @param deliveryDateGuaranteed true if the delivery date is guaranteed for this Rate.
+     * @param estDeliveryDays        The estimated delivery days for this Rate.
+     * @param shipmentId             ID of the shipment of this Rate.
+     * @param carrierAccountId       ID of the carrier account of this Rate.
+     * @param timeInTransit          TimeInTransit object.
+     */
+    public Rate(final String id, final String carrier, final String service, final Float rate, final String currency,
```

Comment:
CheckStyles hates this constructor because we're passing in more than 7 parameters.
I removed it and reworked our RateDeserializer to create it more cleanly.